### ISSUE TITLE
Revise homepage

### ISSUE THREAD

**Issue by ggeisler** [pod4lib/aggregator#470]
The current homepage is out-of-date in several ways (MVP references, deprecated application name) and omits info that could be helpful to new and returning users. A complete redesign of the homepage would be worthwhile and likely not a huge amount of work.

**Comment by ggeisler** [comment]
Below is a proposed redesign of the homepage. In this ticket I'll focus on the new structure of the homepage and the elements that are common to all users. I'll create a separate ticket to focus on the details of what I've labelled section D below, since that seems like a bunch of detailed work that could be done after getting the basic restructuring done (don't want to make this ticket too complicated).

<img width="1196" alt="Screen Shot 2022-03-29 at 2 59 58 PM" src="https://user-images.githubusercontent.com/101482/160715019-15fbd282-b3ea-4505-a14f-30f74106b92f.png">

Above is an example for an organization member who is logged in.

**A**: Re-label the topbar main navigation elements (see additional mockups below for examples not shown above):
  - The current "Organizations" link becomes "Providers" and points to the same page
  - Add a new link "Data" that points to a new page focusing on tasks a data consumer cares most about
  - The current "Activity" link is only shown to users in "Superuser" mode
  - A new link "Users" is only shown to users in "Superuser" mode (goes to the same page the "Manage users" link the user menu goes to)
  - A public/not-logged-in user should not see any of these navigation links
 
**B:** Update the hero title and text.
  - `POD Aggregator`
  - `The POD Aggregator is an open source system to aggregate MARC bibliographic and holdings data for institutions that participate in the Platform for Open Data (POD) initiative.`

Replace the current content between the hero and footer sections with the content labelled "C" and "D" in the mockup above.

**C:** Two equal-width sections that take 50% of the page width. On mobile, they should each take 100% width.
- Use the same border and background colors as used in the site footer
- The headings should be `h2`s but let's try formatting them with the class `.h3` so they're a bit smaller
- If the user is logged in, it would be nice if the entire section of each (not just the heading but the entire block) links to the same place as the "Providers" and "Data" links in the topbar navigation. But if the user is not logged in, those links shouldn't be active.

- `POD Providers`
- `There are currently x data providers. The most recent uploads were made by y1, y2, and y3 on z.` where:
  - `x` = number of data providers
  - `y` = the names of providers who made uploads on the most recent day uploads were made
  - `z` = the date (in human-friendly format, as shown, rather than any of the more compact date-time display formats we currently use in tables) of the most recent day an upload was made

- `POD Data`
- `POD Aggregator currently holds a total of 15,347 files, containing 162,602,513 records, of which 67,890,176 are unique.` where we dynamically show the latest counts for each of the listed entities.

 **D:** This section will vary based on what type of user is logged in. I'll create a separate ticket with details about that.

## Public/Non-logged in user
<img width="1196" alt="Screen Shot 2022-03-29 at 3 01 57 PM" src="https://user-images.githubusercontent.com/101482/160717391-85b83b5c-76bf-49a1-b68e-3b1f9dea6132.png">


## User active in Superuser mode
<img width="1196" alt="Screen Shot 2022-03-29 at 3 09 51 PM" src="https://user-images.githubusercontent.com/101482/160717375-75378b5e-ceb4-4109-8769-8b62e577408a.png">

**Comment by thatbudakguy** [comment]
about the "most recent uploads" display in the providers section — what should we do in cases where:
* all three of the most recent uploads were made by a single institution
* the three most recent uploads occur on different days (e.g. the most recent was today, but the two prior were yesterday)

**Comment by ggeisler** [comment]
@thatbudakguy I know this wasn't clear but my thought wasn't that we were showing the **three** most recent uploads, rather it was whichever providers made an upload on the most recent day there were uploads.

So there could only be one institution, or there could be more than three.

Though that raises the question of what we do if there are a lot of providers that made an upload on the same, most recent day of uploads. Based on current activity that doesn't seem to be something that happens, but it might in the future so if you want to try to handle it now in the code, maybe something like:

> There are currently **14** data providers. The most recent uploads were made by **Duke**, **Penn**, **Princeton** and 4 others on March 23, 2022.

In other words, when more than three providers have uploads on the most recent day:

- List the first three (alphabetical, or by most recent, doesn't really matter since we're just trying convey the sense that people are doing things) and add "and X others"  where X is the total number of providers with recent uploads, minus the 3 we listed explicitly
- (If that seems unnecessarily complicated, it seems fine to just say "and others")

**Comment by ggeisler** [comment]
@thatbudakguy Sure, I think the main goal is just to convey in a concrete way that there has been activity, so you suggestion sounds fine to me.

**Comment by thatbudakguy** [comment]
@ggeisler sorry to revisit this — in working more with the "recent uploads" section in light of our conversation about relative times, i've discovered a couple of things.
- current data patterns mean that on any given day, someone is likely to have uploaded _something_ (and often many things), so the time since last upload is almost never more than one day
- it's a little tricky to get the logic right so it reads "today" if the last upload was within the last 24hr of the user's time zone, but a relative distance like "26 hours ago" in other cases
- if you output the time of the most recent upload, it won't necessarily be true for the other two most recent uploads/uploaders

in light of this my current implementation just does the most recent single upload, so it says e.g.
> The most recent upload was made by Harvard 8 minutes ago. 

i hope this is sufficient to convey a sense of "people are uploading lots of things!" since that time distance should be pretty small in most cases.

**Comment by ggeisler** [comment]
@thatbudakguy No problem, I see what you're saying. Your solution sounds fine. We can see how it feels and adjust if necessary, but I think your solution will accomplish the goal.
